docs/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+1.8.6 (Unreleased)
+----------------------
+- Enh #221: By default, enabled "Mobile" notification types for those enabled for "Web"
```

Comment:
@marc-farre Please use #258 als ID in the Changelog.

Reply:
Done in commit https://github.com/humhub/tasks/pull/258/commits/99af3e48ce2e6f8a71002177b38cd4f6d80b95fc